relayer-cli/src/commands/query/packet.rs

```diff
@@ -254,16 +260,54 @@ impl Runnable for QueryUnreceivedPacketsCmd {
                     "failed to find channel ({}/{}) on chain ({}) with error: {}",
                     opts.port_id,
                     opts.channel_id,
-                    src_chain.config().id,
+                    src_chain.id(),
                     e
                 ))
                 .exit();
             }
         };
 
+        let src_connection_id = match channel.connection_hops().first() {
+            Some(id) => id,
+            None => {
+                return Output::error(format!(
+                    "no connection hops for channel '{}'",
+                    self.src_channel_id
+                ))
+                .exit()
+            }
+        };
+
```

Comment:
I think we can remove this and query channel above.

Reply:
What are the advantages of querying the channel vs pulling the `src_connection_id` from the `channel` data structure already available in memory?